packages/type-storage/src/fromMetadata/v4/createFunction.ts

```diff
@@ -45,7 +45,7 @@ export default function createFunction (section: Text | string, method: Text | s
       assert(!isUndefined(arg) && !isNull(arg), `${meta.name} expects one argument`);
 
       const type = meta.type.asMap.key.toString();
-      const param = createType(type, arg).toU8a(false);
+      const param = createType(type, arg).toU8a();
```

Comment:
Are we sure about this? It will add the prefixes as it stands? (Or did we move from a non-`Bytes` or rather, does it require the length prefixes now?)

Reply:
`isBare` get passed down into subtypes, so `toU8a(false)` should be strictly equivalent to `toU8a()`.